language_service/Cargo.toml

```diff
@@ -24,6 +24,7 @@ enum-iterator = { workspace = true }
 regex-lite = { workspace = true }
 qsc_project = { path = "../compiler/qsc_project", features = ["async"] }
 async-trait = { workspace = true }
+chrono = { workspace = true }
```

Comment:
Do you still need chrono in this crate?

Reply:
I've cleaned up the Cargo.toml files for the language server and the new docs generation module.